src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -44,6 +46,8 @@ public class BlockStoreSignaled : IDisposable
 
         private readonly ISignals signals;
 
+        private SubscriptionToken blockConnectedSubscription;
```

Comment:
Why keep the subscription it snot used.

Reply:
it's used.
A subscribtion is needed to unsubscribe from it (usually in Dispose of the owner)

Reply:
But its not used in this calss is what I mean.

Reply:
actually it is, L255
`this.signals.Unsubscribe(this.blockConnectedSubscription);`